uploadScriptGistsAndGenerateOffchainSects.ts

```diff
@@ -2,32 +2,39 @@ import { SecretsManager, createGist } from "@chainlink/functions-toolkit";
 import { readFileSync } from "fs";
 // Note, you need to switch to the ethers v5.7.2 to run this script.
 import { ethers } from "ethers";
-import { config as envEncConfig } from '@chainlink/env-enc'
+require('dotenv').config();
+import crypto from 'crypto';
+// import { config as envEncConfig } from '@chainlink/env-enc'
 
+console.log('privateKey: ', process.env.PRIVATE_KEY)
+console.log('rpcUrl: ', process.env.RPC_URL)
 const isMainnet = process.env.STAGE === 'mainnet'
 
 console.log(isMainnet ? 'Mainnet' : 'Testnet')
 
-envEncConfig({
-  path: isMainnet ? '/Volumes/TUNNL/encryptedEnvVars/.env.enc.mainnet' : '/Volumes/TUNNL/encryptedEnvVars/.env.enc.testnet'
-});
+// envEncConfig({
+//   path: isMainnet ? '/Volumes/TUNNL/encryptedEnvVars/.env.enc.mainnet' : '/Volumes/TUNNL/encryptedEnvVars/.env.enc.testnet'
+// });
+
+// const sha256 = async (text: string) => {
+//   return Array.from(
+//     new Uint8Array(
+//       await crypto.subtle.digest(
+//         "SHA-256",
+//         new TextEncoder().encode(text)
+//       )
+//     )).map(
+//       (b)=>b.toString(16).padStart(2,"0")
+//     ).join("")
+// }
 
 const sha256 = async (text: string) => {
-  return Array.from(
-    new Uint8Array(
-      await crypto.subtle.digest(
-        "SHA-256",
-        new TextEncoder().encode(text)
-      )
-    )).map(
-      (b)=>b.toString(16).padStart(2,"0")
-    ).join("")
+  return crypto.createHash('sha256').update(text).digest('hex');
```

Comment:
Just as a sanity check, did you compare the before & after hash values to make sure these functions are exactly identical?